apps/common_logging_processors.go

```diff
@@ -88,3 +89,66 @@ func genericAccessLogParser(ctx context.Context, processorType, tag, uid string)
 	c = append(c, mf.Components(ctx, tag, uid)...)
 	return c
 }
+
+// TODO: rename to genericAccessLogParser once the old version is removed
+// genericAccessLogParserAsInternalLoggingProcessor is an internal logging processor that parses access logs.
+// It will eventually replace genericAccessLogParser, as it returns a slice of InternalLoggingProcessor rather than fluentbit.Component, making it more flexible.
+func genericAccessLogParserAsInternalLoggingProcessor(ctx context.Context, processorType, tag, uid string) []confgenerator.InternalLoggingProcessor {
```

Comment:
Ohh! I missed this before, `tag` and `uid` are fluent-bit specific parameters which are not used in the new implementation. They can be removed now.